### PR TITLE
Updating FAQ / funding disclosure

### PR DESCRIPTION
**Pull request by shiffman**
This pull request updates the FAQ to reflect the Steering Committee membership and other details in preparation for launch (#27) I've also chosen to add the context about project funding to this document rather than create a separate one. I wonder if something like this should make its way onto ml5js.org as well?

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,37 +1,70 @@
...
+Second, it makes revising or changing the license easier. The ml5.js license is an experiment in ethical licensing. If it turns out we made a horrible mistake, we only have to wait for three years until it disappears.
 
 ## Who is on the Code of Conduct Committee and how is the Committee selected?
 
-The Code of Conduct Committee is responsible for maintaining the Code of Conduct and reviewing accused violations of the Code of Conduct.  The Committee selection process will be finalized as part of the community review of the proposed license.  Once finalized, the will be incorporated into the Code of Conduct and current members will be listed in the document. 
+The Code of Conduct Committee is responsible for maintaining the Code of Conduct as well as deciding if a given use of ml5.js violates the Code of Conduct. It can make those decisions in response to a specific request, or through an investigation it initiates directly.
+
+The Code of Conduct Committee’s members are:
```
Comment (joeyklee) [inline FAQ.md:39]: Suggestion:
* Thinking about maintainability -- we currently list these members in 2 different places -- the FAQ and in the CoC. Maybe we have a `COMMITTEE.md` that we link to?

File: FAQ.md [hunk 2]
```diff
@@ -1,37 +1,70 @@
...
+
+### Processing Foundation
+
+The ml5.js library, though now broadly applicable in native JavaScript, was originally intended as a “plug-in” to add machine learning capabilities to the general purpose creative coding library p5.js, maintained and developed by The Processing Foundation. The Processing Foundation has contributed to the development of ml5.js in a myriad of ways via the intersection of the ml5.js and creative coding communities. Processing Foundation fellowships and other community events have featured and provided support for ml5.js.
+
+### COSA
+
+The ml5.js project has received support from the [Clinic for Open-Source Arts](https://www.du.edu/ahss/opensourcearts/).
```
Comment (joeyklee) [inline FAQ.md:70]: Addition:
* we should also include: 

```
### Frank-Ratchye STUDIO for Creative Inquiry

The ml5.js project has received support from the [Frank-Ratchye STUDIO for Creative Inquiry](https://studioforcreativeinquiry.org/)

```

**Review by joeyklee — CHANGES_REQUESTED** [review]
Just a few notes:
* Added 1 suggestion about `Committee`
* 1 blocking necessary addition on funding/support from STUDIO

**Comment by mwweinberg** [comment]
Pending Joey's stuff this looks good to me

File: FAQ.md (hunk 2 above)
Comment (shiffman) [reply FAQ.md:70]: Added with https://github.com/ml5js/Code-of-Conduct/pull/28/commits/640a654345bf1b6a6708bbcabb84ed67a9b44476

File: FAQ.md (hunk 1 above)
Comment (shiffman) [reply FAQ.md]: Addressed in: https://github.com/ml5js/Code-of-Conduct/pull/28/commits/662cce7187c3fbf6ee4741e5eafffe44b4050cc4

**Review by bomanimc — APPROVED** [review]
Nice! Looks good to me!

**Review by mwweinberg — APPROVED** [review]
Looks good to go!

**Pull request merged**